### ISSUE TITLE
Headslugs and Nanites

### ISSUE THREAD

**Issue by LordNoodlez** [tgstation/tgstation#50409]
Issue reported from  Round ID: [133945](https://scrubby.melonmesa.com/round/133945) (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])
Reporting client version: 513.1515

Somehow as a changeling headslug, I got nanites and was ghosted out of the round because of how nanite defib works. I found this out by trying to plant an egg in a dead guy a nightmare was so kind enough to bring me. After the headslug death before resurrection, a prompt came up for revival of some sort. I didn't read it since I had never actually done headslug transformation before. The prompt was for nanite revival by defib. Once I accepted, I was shunted back into my headslug body and promptly died from headslug death syndrome, and the egg never hatched. I don't know how repeatable this is, but headslugs generally shouldn't have nanites, as i did not have nanites as a human person, so they kind of came out of thin air.

tl;dr: Headslugs can get nanites that prevent them from resurrecting as a human person.

**Comment by XDTM** [comment]
i didn't expect defib to work on simple mobs in the first place

**Comment by adamkad1** [comment]
> i didn't expect defib to work on simple mobs in the first place

You mean i dont have to carry a strange reagent gas death grenade as a gorilla?
